BuyPolish/Pola/UI/ProductSearch/Keyboard/BPKeyboardTextView.m

```diff
@@ -0,0 +1,59 @@
+#import "BPKeyboardTextView.h"
+
+static CGFloat const kCornerRadious = 2.0;
```

Comment:
shouldn't it have context prefix, like `kKeyboardTextViewCornerRadius`? also there is a typo with `Radious`